From 66189546403e5476c560bdbad89d24c3df87b172 Mon Sep 17 00:00:00 2001
From: Yue Ni <niyue.com@gmail.com>
Date: Tue, 19 Jan 2021 17:03:02 +0800
Subject: [PATCH] Fix hyperscan compilation issue for
 https://github.com/intel/hyperscan/issues/292, avoid the compilation error
 reporting 'undefined reference to `avx2_snprintf`'.

---
 cmake/build_wrapper.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/build_wrapper.sh b/cmake/build_wrapper.sh
index 1962813f..895610c0 100755
--- a/cmake/build_wrapper.sh
+++ b/cmake/build_wrapper.sh
@@ -17,7 +17,7 @@ KEEPSYMS=$(mktemp -p /tmp keep.syms.XXXXX)
 LIBC_SO=$("$@" --print-file-name=libc.so.6)
 cp ${KEEPSYMS_IN} ${KEEPSYMS}
 # get all symbols from libc and turn them into patterns
-nm -f p -g -D ${LIBC_SO} | sed -s 's/\([^ ]*\).*/^\1$/' >> ${KEEPSYMS}
+nm -f p -g -D ${LIBC_SO} | sed -s 's/\([^ @]*\).*/^\1$/' >> ${KEEPSYMS}
 # build the object
 "$@"
 # rename the symbols in the object
